programs/bpf/tests/programs.rs

```diff
@@ -1524,7 +1524,7 @@ fn assert_instruction_count() {
             ("solana_bpf_rust_noop", 366),
             ("solana_bpf_rust_param_passing", 146),
             ("solana_bpf_rust_rand", 469),
-            ("solana_bpf_rust_sanity", 52054),
+            ("solana_bpf_rust_sanity", 52232),
```

Comment:
https://discord.com/channels/428295358100013066/560503042458517505/1066922391525142579

:man_shrugging:

Reply:
i looked into this a bit.  https://discord.com/channels/428295358100013066/560503042458517505/1067210880204406875

i'd like to think we'll be fine because there's little reason to instantiate a `Pubkey` at runtime in a program.  they should all get built via deserialization or declared via `pubkey!()` et. al macros.